docs/source/models.rst

```diff
@@ -2670,6 +2776,77 @@ avoid failed inference requests.
 
 \***** If using `pyfunc` in MLflow Model Serving for realtime inference, the raw audio in bytes format must be base64 encoded prior to submitting to the endpoint. String inputs will be interpreted as uri locations.
 
+Using inference_config and model signature parameters for `transformers` inference
+""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""
+For `transformers` inference, there are two ways to pass in additional arguments to the pipeline.
+
+* Use inference_config when saving/logging the model
+* Specify parameters at inference time when calling `predict()`
+
+.. note::
+    Model signature parameters passed in to `predict` function will override the values in inference_config.
+    This functionality is not supported for model serving yet, but it will be supported in the short future.
```

Comment:
I think we can remove this line, since we're supporting this capability in model serving as part of this PR, right?

```suggestion
```